src/test/java/com/iggroup/universityworkshopmw/domain/services/ClientServiceTest.java

```diff
@@ -19,60 +18,95 @@ public void setUp() {
 
    @Test
    public void storeNewClient_assignsUniqueIdPerAddition() {
-      Client client1 = createClient("", "userName1", 0);
-      Client client2 = createClient("", "userName2", 0);
+      Client client1 = createClient("", "userName1", 0, 0);
+      Client client2 = createClient("", "userName2", 0, 0);
 
       Client returnClient1 = clientService.storeNewClient(client1);
       Client returnClient2 = clientService.storeNewClient(client2);
 
-      assertNotNull(returnClient1.getId());
-      assertNotNull(returnClient2.getId());
-      assertNotEquals(returnClient1.getId(), returnClient2.getId());
+      assertThat(returnClient1.getId()).isNotNull();
```

Comment:
We seem to be using a lot of different assertions. What makes these assertions better than `AssertEquals/AssertNotNull` from junit?

Reply:
This will be done in another story in the backlog -> should be simple to change all to use one. With assertj it allows us to do simple assertions, like with lists, a lot easier to assert them :) Backlog item is to rewrite all assertions to use assertj instead of generic junit assert